app/components/data_protection_confirmation_banner_component.rb

```diff
@@ -13,26 +15,62 @@ def initialize(user:, organisation: nil)
   def display_banner?
     return false unless FeatureToggle.new_data_protection_confirmation?
     return false if user.support? && organisation.blank?
+    return true if org_without_dpo?
+
+    !org_or_user_org.data_protection_confirmed?
+  end
+
+  def header_text
+    if org_without_dpo?
+      "To create logs your organisation must state a data protection officer. They must sign the Data Sharing Agreement."
+    elsif user.is_dpo?
+      "Your organisation must accept the Data Sharing Agreement before you can create any logs."
+    else
+      "Your data protection officer must accept the Data Sharing Agreement on CORE before you can create any logs."
+    end
+  end
 
-    !DataProtectionConfirmation.exists?(
-      organisation: org_or_user_org,
-      confirmed: true,
-    )
+  def second_row
```

Comment:
Can we rename this to something else? Like banner_text/banner_detail/banner_content etc?